savant/deepstream/pipeline.py

```diff
@@ -108,6 +108,7 @@ def __init__(
         init_telemetry(name, telemetry)
 
         output_frame = kwargs.get('output_frame')
+        self._pass_through_mode = bool(output_frame) and output_frame['codec'] == 'copy'
```

Comment:
unnecessary bool()

Reply:
We need it to be strictly bool since the value is set as a property of `zeromq_source_bin`.